src/service/mobile_config.proto

```diff
@@ -95,6 +95,7 @@ enum admin_key_role {
 enum network_key_role {
   mobile_carrier = 0;
   mobile_router = 1;
+  mobile_pcs = 2;
```

Comment:
maybe comment these so they have explanations what they mean? this will also need a complimentary migration in the oracles to add a `pcs` to the database key_role enum.